Update Getting Started in order to keep its updated (#57)
operator-framework/getting-started: README.md

}
type MemcachedStatus struct {
    // Nodes are the names of the memcached pods
    Nodes PodList `json:"nodes"`
}

// PodList is a list of names of pods
type PodList struct {

> 	Names []string

Author: ```suggestion
    Names []string
```

}

func NewPodList(value []string) PodList {

> 	p := PodList{Names: value}

Author: ```suggestion
    p := PodList{Names: value}
```

> 	return p

Author: ```suggestion
    p := PodList{Names: value}
    return p
```

## Manage the operator using the Operator Lifecycle Manager

> NOTE: This section of the Getting Started Guide is out-of-date.
> We're working on some improvements to Operator SDK to streamline

> > the experience of using OLM. For further information see, for example, the [proposal][sdk-integration-with-olm-doc]. 

Reviewer: ```suggestion
> the experience of using OLM. For further information see, for example, this enhancement [proposal][sdk-integration-with-olm-doc]. 
```

> > In the meantime, you might find the following documentation helpful: 

Reviewer: ```suggestion
```

Author: I would prefer to keep this part because if we remove it and just keep the note then users may not check the following steps since they will assume that they will not work.

The previous section has covered manually running an operator. In the next sections, we will explore using the [Operator Lifecycle Manager][operator_lifecycle_manager] (OLM) which is what enables a more robust deployment model for operators being run in production environments.

OLM helps you to install, update, and generally manage the lifecycle of all of the operators (and their associated services) on a Kubernetes cluster. It runs as an Kubernetes extension and lets you use `kubectl` for all the lifecycle management functions without any additional tools.

> **NOTE:** You can check many public operator projects available to be used with OLM in the [OperatorHub.io][operator-hub-io]. 

Reviewer: ```suggestion
**NOTE:** Various public, OLM-ready operator projects are available at [operatorhub.io][operator-hub-io]. 
```

**NOTE:** Various public, OLM-ready operator projects are available at [operatorhub.io][operator-hub-io]. 

### Generate an operator manifest

The first step to leveraging OLM is to create a [Cluster Service Version][csv_design_doc] (CSV) manifest. An operator manifest describes how to display, create and manage the application, in this case memcached, as a whole. It is required for OLM to function.

The Operator SDK CLI can generate CSV manifests via the following command:

```console
$ operator-sdk olm-catalog gen-csv --csv-version 0.0.1 --update-crds
```

Several fields must be updated after generating the CSV. See the CSV generation doc for a list of [required fields][csv-fields], and the memcached-operator [CSV][memcached_csv] for an example of a complete CSV.

**NOTE:** You are able to preview and validate your CSV manifest syntax in the [operatorhub.io CSV Preview][operator-hub-io-preview] tool.

### Testing locally

The next step is to ensure your project deploys correctly with OLM and runs as expected. Follow this [testing guide][testing-operators] to deploy and test your operator.

> **NOTE:** Also, you may would like to check a feature to build the bundle in the [OperatorHub.io][operator-hub-i-bundle].

Reviewer: ```suggestion
**NOTE:** check out operatorhub.io's [bundle build tool][operator-hub-io-bundle] as an alternative to using `operator-sdk olm-catalog gen-csv`.
```

> **NOTE:** Also, you may would like to check a feature to build the bundle in the [OperatorHub.io][operator-hub-i-bundle].

Author: ```suggestion
**NOTE:** Also, check out operatorhub.io's [bundle build tool][operator-hub-io-bundle] as an alternative to using `operator-sdk olm-catalog gen-csv`.
```

```console
$ operator-sdk olm-catalog gen-csv --csv-version 0.0.1
```

> Then, after generate the CSV file will be required update it with the required information over the memcached-project which could not be done automatically by the tool. See an example of an [Memcached CSV][memcached_csv] file filled.

Reviewer: ```suggestion
Several fields must be updated after generating the CSV. See the CSV generation doc for a list of [required fields][csv-fields], and the memcached-operator [CSV][memcached_csv] for an example of a complete CSV.
```

And make a link ref below for `csv-fields: https://github.com/operator-framework/operator-sdk/blob/master/doc/user/olm-catalog/generating-a-csv.md#csv-fields`

> **NOTE:** You are able to preview and validate your CSV file syntaxe in [OperatorHub.io CSV Preview][operator-hub-io-preview].

Reviewer: ```suggestion
**NOTE:** You are able to preview and validate your CSV manifest syntax in the [operatorhub.io CSV Preview][operator-hub-io-preview] tool.
```

### Testing locally

> The next step to check your project working with OLM will be generate its bundle. Following the [Testing your Operator with Operator Framework][testing-operators] to know how to perform it and test its integration.

Reviewer: ```suggestion
The next step is to ensure your project deploys correctly with OLM and runs as expected. Follow this [testing guide][testing-operators] to deploy and test your operator.
```

The bundle is created when you run `gen-csv --update-crds`. I suggest adding a note about that flag when `gen-csv` is run above.

> **NOTE:** Also, you may would like to check a feature to build the bundle in the [OperatorHub.io][operator-hub-i-bundle].

Reviewer: Do we want to make note of this, or direct users to only use `gen-csv`? @dmesser

Author: I think that would be nice to give the opportunity/info in order to allow users to check this feature as well.

> ### Ensuring the quality

Reviewer: ```suggestion
### Promoting operator standards
```

> It is recommended use the `operator-sdk scorecard` in order to check project quality as its integration with OLM. For further information see [scorecard documentation][scorecard-doc].

Reviewer: ```suggestion
We recommend running `operator-sdk scorecard` against your operator to see whether your operator's OLM integration follows best practices. For further information on running the scorecard and results, see the [scorecard documentation][scorecard-doc].

**NOTE:** the scorecard is undergoing changes to give informative and helpful feedback. The original scorecard functionality will still be available while and after changes are made.
```

[result_go_doc]: https://godoc.org/github.com/kubernetes-sigs/controller-runtime/pkg/reconcile#Result
[doc_client_api]: https://github.com/operator-framework/operator-sdk/blob/master/doc/user/client.md
[repo_sdk_samples_memcached]: https://github.com/operator-framework/operator-sdk-samples/tree/master/memcached-operator/
[mailing_list]: https://groups.google.com/forum/#!forum/operator-framework
[memcached_csv]: https://github.com/operator-framework/operator-sdk/blob/master/test/test-framework/deploy/olm-catalog/memcached-operator/0.0.3/memcached-operator.v0.0.3.clusterserviceversion.yaml
[testing-operators]: https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md
[sdk-integration-with-olm-doc]: https://github.com/operator-framework/operator-sdk/blob/master/doc/proposals/sdk-integration-with-olm.md
[operator-hub-io]: https://operatorhub.io/ 

> [operator-hub-io-preview]: https://operatorhub.io/preview.

Reviewer 2: ```suggestion
[operator-hub-io-preview]: https://operatorhub.io/preview
```

> [operator-hub-i-bundle]: https://operatorhub.io/bundle.

Reviewer 2: ```suggestion
[operator-hub-i-bundle]: https://operatorhub.io/bundle
```
